backend/app/assets/stylesheets/spree/backend/components/_navigation.scss

```diff
@@ -2,6 +2,49 @@ $padding-x-navbar: 26px;
 $padding-y-navbar: 13px;
 $padding-y-navbar-submenu: 9px;
 
+.menu-button-icon {
+  width: 34px;
+  height: 32px;
+  margin-right: 10px;
+  background: $color-btn-bg;
+  color: $color-btn-text;
+  border-radius: 4px;
+  text-align: center;
+  cursor: pointer;
+  display: none;
+  &:hover {
+    background: $color-btn-hover-bg;
+  }
+  @include media($tablet) {
+    display: block;
+  }
+
+  i {
+    padding-top: 7px;
+    color: $color-btn-text;
+    font-size: 18px;
+  }
+}
+
+#menu-button {
+  display: none;
+
+  &:checked ~ .admin-wrapper {
+    overflow: hidden;
+
+    .admin-nav {
+      transform: translateX(0);
+    }
+
+    .admin-content  {
+      transform: translateX($width-sidebar);
+      @include media($large-device) {
+        transform: none;
+      }
+    }
+  }
+}
+
```

Comment:
I would not consider this a bug, but I also don't persist on this animation at all. It was just an idea for a little nicer UX.